.circleci/docker/build_docker.sh

```diff
@@ -49,6 +50,13 @@ fi
 # Only push if `DOCKER_SKIP_PUSH` = false
 if [ "${DOCKER_SKIP_PUSH:-true}" = "false" ]; then
   docker push "${image}:${tag}"
+
+  if [ "${BUILD_TYPE}" = "release" ]; then
```

Comment:
Where do we expect this to get populated?

As well we should probably add a default for this like so:

```suggestion
  if [ "${BUILD_TYPE:-}" = "release" ]; then
```

Reply:
I think we can probably probably do something similar to what we do with our `pytorch/builder` workflows where we set an env variable like so (probably name it something like `PUSH_GHCR_IMAGE`):

https://github.com/pytorch/builder/blob/4c1b45bbc550e4b6d45853627fe53cf64b1de71b/.github/workflows/build-conda-images.yml#L21

Reply:
I added the env variable that can be set as the action's input.